omod/src/test/java/org/openmrs/web/controller/user/UserFormControllerTest.java

```diff
@@ -22,27 +24,50 @@
 import org.springframework.web.context.request.ServletWebRequest;
 import org.springframework.web.context.request.WebRequest;
 
+
 /**
  * Tests the {@link oldUserFormController} class.
  */
 public class UserFormControllerTest extends BaseModuleWebContextSensitiveTest {
 	
+	protected static final String TEST_DATA = "org/openmrs/web/controller/include/UserFormControllerTest.xml";
+	
 	@Autowired
 	private UserFormController controller;
 	
 	/**
-	 * @see UserFormController#handleSubmission(WebRequest,HttpSession,String,String,String,null,User,BindingResult)
-	 * 
+	 * @see UserFormController#handleSubmission(WebRequest,HttpSession,String,String,String,null, String, User,BindingResult)
+	 *
 	 */
 	@Test
-	@Verifies(value = "should work for an example", method = "handleSubmission(WebRequest,HttpSession,String,String,String,null,User,BindingResult)")
+	@Verifies(value = "should work for an example", method = "handleSubmission(WebRequest,HttpSession,String,String,String,null, String, User,BindingResult)")
 	public void handleSubmission_shouldWorkForAnExample() throws Exception {
 		WebRequest request = new ServletWebRequest(new MockHttpServletRequest());
 		User user = controller.formBackingObject(request, null);
 		user.addName(new PersonName("This", "is", "Test"));
 		user.getPerson().setGender("F");
 		controller.handleSubmission(request, new MockHttpSession(), new ModelMap(), "", "Save User", "pass123", "pass123", null,
-		    null, null, new String[0], "true", user, new BindException(user, "user"));
+				null, null, new String[0], "true", null, user, new BindException(user, "user"));
+	}
+	
+	/**
+	 * @see UserFormController#handleSubmission(WebRequest,HttpSession,String,String,String,null, String, User,BindingResult)
+	 *
+	 */
+	@Test
+	@Verifies(value = "Creates Provider Account when Provider Account Checkbox is selected", method = "handleSubmission(WebRequest,HttpSession,String,String,String,null, String, User,BindingResult)")
+	public void handleSubmission_createUserProviderAccountWhenProviderAccountCheckboxIsSelected() throws Exception {
+		executeDataSet(TEST_DATA);
```

Comment:
FYI, you could've used one of the test users in the standardDataset.xml file without creating a new one